### ISSUE TITLE
linode-cli linodes create fails with AttributeError: CLIArg instance has no attribute 'required'

### ISSUE THREAD

**Issue by snitko** [linode/linode-cli#30]
```
~ $ linode-cli linodes create --help
linode-cli linodes create
Create Linode

Arguments:
Traceback (most recent call last):
  File "/usr/local/bin/linode-cli", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python2.7/dist-packages/linodecli/__init__.py", line 193, in main
    for arg in sorted(operation.args, key=lambda s: not s.required):
  File "/usr/local/lib/python2.7/dist-packages/linodecli/__init__.py", line 193, in <lambda>
    for arg in sorted(operation.args, key=lambda s: not s.required):
AttributeError: CLIArg instance has no attribute 'required'
```
I'm on Ubuntu 18.04, Pythin 2.7 as can be seen, linode-cli installed with pip.

**Comment by gaswirth** [comment]
Hey there,

I’m having the same issue, with `linode-cli stackscripts update`. I’ve tried both on Ubuntu 16.04 and OS X, with identical issues. Installed with pip on both. Output follows:

`Traceback (most recent call last):
  File "/home/gaswirth/.local/bin/linode-cli", line 11, in <module>
    sys.exit(main())
  File "/home/gaswirth/.local/lib/python2.7/site-packages/linodecli/__init__.py", line 209, in main
    cli.handle_command(parsed.command, parsed.action, args)
  File "/home/gaswirth/.local/lib/python2.7/site-packages/linodecli/cli.py", line 413, in handle_command
    result = self.do_request(operation, args)
  File "/home/gaswirth/.local/lib/python2.7/site-packages/linodecli/cli.py", line 323, in do_request
    parsed_args = operation.parse_args(args)
  File "/home/gaswirth/.local/lib/python2.7/site-packages/linodecli/operation.py", line 111, in parse_args
    parser.add_argument('--'+arg.path, metavar=arg.name,
AttributeError: CLIArg instance has no attribute 'path'`

**Comment by Dorthu** [comment]
This appears to be related to #27 and has something to do with the python2 version of the CLI.  I'm working on a fix, in the meantime you can work around this by running it in python3:

```bash
pip uninstall linode-cli
pip3 install linode-cli
linode-cli linodes create --help # this should work
```

I'll update this issue when the python2 version is fixed.

**Comment by Dorthu** [comment]
This should be fixed in `2.0.1b0` - run a `pip install --upgrade linode-cli` to get the latest version.  Let me know if it works so I can close this issue.
